data/cmdvartab

```diff
@@ -106,14 +117,31 @@ VARDESC battery.charger.status "Battery charger status"
 
 VARDESC ambient.temperature "Ambient temperature (degrees C)"
 VARDESC ambient.temperature.alarm "Ambient temperature alarm is active"
+VARDESC ambient.temperature.status "Ambient temperature status relative to the configured thresholds"
 VARDESC ambient.temperature.alarm.maximum "Maximum allowed ambient temperature"
 VARDESC ambient.temperature.alarm.minimum "Minimum allowed ambient temperature"
 VARDESC ambient.temperature.alarm.enable "Enable ambient temperature alarm"
+VARDESC ambient.temperature.low "Temperature threshold low"
```

Comment:
Should the string descriptions be followed by "(degrees C)" like some other lines around here?